contracts/contribute/RecovererContributeManager.sol

```diff
@@ -162,8 +156,11 @@ contract RecovererContributeManager {
         // Record refund has been made
         mapAccountRefunds[msg.sender] = true;
 
+        IService.TokenSecurityDeposit memory tokenSecurityDeposit =
```

Comment:
This was a bug :). Should return the token security deposit from `ServiceRegistryTokenUtility` and not the mocking 1 wei from the `ServiceRegistry`